tests/expectations/tests/bitfield_align_2.rs

```diff
@@ -4,6 +4,7 @@
     non_camel_case_types,
     non_upper_case_globals
 )]
+#![cfg(not(target_os = "windows"))]
```

Comment:
this type of "hack" works, but maybe somehow all/specific tests can be annotated with `#[cfg_attr(<cfg>, ignore)]` at will